docs/templates/plugins.rst.j2

```diff
@@ -30,8 +30,10 @@
             {% else %}
     Default: {% for default_item in actual_default %}``{{ default_item.pattern | default(default_item) }}``{% if not loop.last %}, {% endif %}{% endfor %}
             {% endif %}
+        {% elif actual_default.__class__.__name__ == 'CheckResultInterpret' %}
```

Comment:
Ugly workaround.  
@happz Do you have any cleaner solution to point me to please? I spent way more time than I'd like to admit trying to solve this.

Reply:
Yeah, this is not very nice :/ What fields forced you to take this road? `result` is the new one, but it's a string, like `how`, so I don't immediately recall what might be a problem. What's logged by the script?

Reply:
Nevermind, I can try it out.

Reply:
It says `CheckResultInterpret.RESPECT` is the value. I've tried to add `exporter=lambda result: result.value` to Check.result field, but to no avail.

Reply:
Yep, got it, it's an enum, I suppose it's the first one in plugins, and we can solve it once and for all :)

```diff
diff --git a/docs/scripts/generate-plugins.py b/docs/scripts/generate-plugins.py
index a37d83c1..9368f731 100755
--- a/docs/scripts/generate-plugins.py
+++ b/docs/scripts/generate-plugins.py
@@ -1,6 +1,7 @@
 #!/usr/bin/env python3
 
 import dataclasses
+import enum
 import sys
 import textwrap
 from typing import Any
@@ -106,6 +107,12 @@ def container_intrinsic_fields(container: ContainerClass) -> list[str]:
     return field_names
 
 
+def is_enum(value: Any) -> bool:
+    """ Find out whether a given value is an enum member """
+
+    return isinstance(value, enum.Enum)
+
+
 def _create_step_plugin_iterator(registry: tmt.plugins.PluginRegistry[tmt.steps.Method]):
     """ Create iterator over plugins of a given registry """
 
@@ -184,6 +191,7 @@ def main() -> None:
         STEP=step_name,
         PLUGINS=plugin_generator,
         REVIEWED_PLUGINS=REVIEWED_PLUGINS,
+        is_enum=is_enum,
         container_fields=tmt.utils.container_fields,
         container_field=tmt.utils.container_field,
         container_ignored_fields=container_ignored_fields,
diff --git a/docs/templates/plugins.rst.j2 b/docs/templates/plugins.rst.j2
index 5b99149c..5969cc4c 100644
--- a/docs/templates/plugins.rst.j2
+++ b/docs/templates/plugins.rst.j2
@@ -30,7 +30,7 @@
             {% else %}
     Default: {% for default_item in actual_default %}``{{ default_item.pattern | default(default_item) }}``{% if not loop.last %}, {% endif %}{% endfor %}
             {% endif %}
-        {% elif actual_default.__class__.__name__ == 'CheckResultInterpret' %}
+        {% elif is_enum(actual_default) %}
     Default: ``{{ actual_default.value }}``
         {% else %}
             {% set _ = LOGGER.warn("%s/%s.%s: could not render default value, '%s'" | format(STEP, plugin_id, field_name, actual_default), shift=0) %}
```

Reply:
Nice, thanks.